proxy-server/src/server.test.js

```diff
@@ -148,7 +148,13 @@ describe('Testing Proxy Server...', async () => {
                           .send(request1);
       recordId = res.body.records[0].id;
     });
-    it('Should create Contacts (POST)', async () => {
+    it('should POST a new entry', async function () {
+      const res = await proxyServer()
+                    .post('/')
+                    .set('Host', passConduit)
+                    .send(request1);
+      expect(res.status).to.equal(200);
+      expect(res.body).to.haveOwnProperty('records');
```

Comment:
What the distinction between this test case and the previous one which is testing for ability to `create an arbitrary record`?

Reply:
that one is a `before` hook so that the `GET` by ID, `PUT` and `PATCH` can work in parallel without having to wait for this `POST` test to pass so that they can run.